ci/verify-chefdk.sh

```diff
@@ -2,6 +2,9 @@
 
 set -evx
 
+# Cleanning up some cruft from previous tests
+sudo find /tmp -name 'chef-dk*' | sudo xargs rm -rf
```

Comment:
Didn't @schisamo not want us using `sudo` as much? I think it makes sense here to cleanup cruft that was generated by another `sudo` command but I would like his opinion